spec/requests/providers/proceeding_merits_task/specific_issue_controller_spec.rb

```diff
@@ -0,0 +1,96 @@
+require "rails_helper"
+
+module Providers
+  module ProceedingMeritsTask
+    RSpec.describe SpecificIssueController do
+      let(:legal_aid_application) { create(:legal_aid_application, :with_proceedings, explicit_proceedings: %i[da001 se004]) }
+      let(:smtl) { create(:legal_framework_merits_task_list, :da001_and_se004, legal_aid_application:) }
+      let(:proceeding) { legal_aid_application.proceedings.find_by(ccms_code: "DA001") }
+      let(:proceeding_two) { legal_aid_application.proceedings.find_by(ccms_code: "SE004") }
+      let(:provider) { legal_aid_application.provider }
+
+      before { allow(LegalFramework::MeritsTasksService).to receive(:call).with(legal_aid_application).and_return(smtl) }
+
+      describe "GET /providers/merits_task_list/:id/specific_issue" do
+        subject(:get_specific_issue) { get providers_merits_task_list_specific_issue_path(proceeding) }
+
+        context "when the provider is not authenticated" do
+          before { get_specific_issue }
+
+          it_behaves_like "a provider not authenticated"
+        end
+
+        context "when the provider is authenticated" do
+          before do
+            login_as provider
+            get_specific_issue
+          end
+
+          it "returns http success" do
+            expect(response).to have_http_status(:ok)
+          end
+        end
+      end
+
+      describe "PATCH providers/merits_task_list/:id/specific_issue" do
+        subject(:patch_specific_issues) { patch providers_merits_task_list_specific_issue_path(proceeding_two), params: params.merge(submit_button) }
+
+        let(:details) { Faker::Lorem.paragraph }
+        let(:confirmed) { true }
+        let(:params) do
+          {
+            proceeding_merits_task_specific_issue: {
+              details:,
+              confirmed: confirmed.to_s,
+            },
+          }
+        end
+
+        context "when the provider is authenticated" do
+          before do
+            login_as provider
+            patch_specific_issues
+          end
+
+          context "with Continue button pressed" do
+            let(:submit_button) do
+              {
+                continue_button: "Continue",
+              }
+            end
+
+            it "updates the record" do
+              expect(proceeding_two.specific_issue.reload.details).to eq(details)
+              expect(proceeding_two.specific_issue.reload.confirmed).to eq(confirmed)
```

Comment:
We could assert this in a single expectation if you wanted, but I don't really mind.

```suggestion
              expect(proceeding_two.specific_issue.reload).to have_attributes(
                details:,
                confirmed:,
              )
```